src/index.js

```diff
@@ -16,7 +16,7 @@ const { channelRouter } = require("./routes/channels/channels.router");
 const app = express();
 
 // The port the express app will listen on
-const port = process.env.PORT || 8081;
+const port = process.env.PORT;
```

Comment:
this means `PORT` **must** be set on in your environment variables...is that intentional?

Reply:
Yes! We have the variable set in the .env 👍